pandas/core/internals/blocks.py

```diff
@@ -1691,7 +1690,7 @@ def _nanpercentile(values, q, axis, **kw):
             if self.ndim == 1:
                 ax = Float64Index([qs])
             else:
-                ax = mgr.axes[0]
+                ax = axes[0]
```

Comment:
This is the only place where the passed `mgr` is ever used.  I think we can get `axes` out too, but am holding off on that.